org.eclipse.scout.sdk.core.s/src/main/java/org/eclipse/scout/sdk/core/s/apidef/Scout22Api.java

```diff
@@ -14,11 +14,29 @@
 
 @MaxApiLevel(22)
 @SuppressWarnings({"squid:S2176", "squid:S00118", "squid:S00100", "findbugs:NM_METHOD_NAMING_CONVENTION", "squid:S2166"}) // naming conventions
-public interface Scout22Api extends IScoutApi, IScoutChartApi, IScout22DoApi {
+public interface Scout22Api extends IScoutApi, IScout22Api, IScoutChartApi, IScout22DoApi {
```

Comment:
Scout22010Api should implement IScout22Api instead, right?

And: maybe merge IScout22DoApi with IScout22Api now?